zerver/lib/test_helpers.py

```diff
@@ -482,24 +482,7 @@ def find_pattern(pattern: Any, prefixes: List[str]) -> None:
             "confirmation_key/",
             "node-coverage/(?P<path>.+)",
             "docs/(?P<path>.+)",
-            "help/add-custom-emoji",
-            "help/configure-who-can-add-custom-emoji",
-            "help/change-the-topic-of-a-message",
-            "help/configure-missed-message-emails",
-            "help/community-topic-edits",
-            "help/about-streams-and-topics",
-            "help/delete-a-stream",
-            "help/add-an-alert-word",
-            "help/change-notification-sound",
-            "help/configure-message-notification-emails",
-            "help/disable-new-login-emails",
-            "help/test-mobile-notifications",
-            "help/troubleshooting-desktop-notifications",
-            "help/web-public-streams",
             "for/working-groups-and-communities/",
```

Comment:
This one is a redirect too; it just was in a different place in the file.

Reply:
Added in updated version, which extends this new system to include all redirects that were in `zproject.urls.py`.

Reply:
**Update**: Moved to 2nd commit because this is a corporate landing page URL.